spec/controllers/api/collectivities_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe Api::CollectivitiesController, type: :controller do
+  describe "GET index" do
```

Comment:
```suggestion
  describe "GET index" do
    subject(:body) do
      get :index 
      response.parsed_body
    end
```

Et en dessous tu manipules directement le `body`